src/main/io/serial_cli.c

```diff
@@ -998,6 +998,14 @@ static void cliWrite(uint8_t ch);
 static bool cliDumpPrintf(uint8_t dumpMask, bool equalsDefault, const char *format, ...);
 static bool cliDefaultPrintf(uint8_t dumpMask, bool equalsDefault, const char *format, ...);
 
+static void cliBreak(const char *str, uint8_t len)
```

Comment:
`cliRepeat` would be clearer I think. Also, should probably be inside `#ifndef CLI_MINIMAL_VERBOSITY`.